client/src/components/Invite/Invite.scss

```diff
@@ -0,0 +1,25 @@
+@import 'styles/variables';
+@import 'styles/helpers';
+
+.Invite {
+  h2 {
+    margin-bottom: $spacing-small;
+  }
+  &__LinkBlock {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: $spacing-small;
+    align-items: center;
+    [type='text'] {
+      flex-grow: 3;
```

Comment:
Instead of this, apply: `flex-basis: 75%`. It will then takes a width of 75% of the flex parent. I think it's easier to understand. `flex-grow` is vague.